include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -313,6 +325,89 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
   return overallObjective;
 }
 
+// Transforms the candidate into the given bounds.
+template<typename SelectionPolicyType>
+template<typename ElemType, typename BaseMatType>
+void CMAES<SelectionPolicyType>::BoundaryTransform(BaseMatType& matrix)
+{
+  double diff = (upperBound - lowerBound) / 2.0;
```

Comment:
Looks like we could use `const double` here, for most of the parameter.

Reply:
It is done.